requirements/requirements.txt

```diff
@@ -3,8 +3,6 @@ huggingface_hub>=0.24
 numba
 numpy>=1.22
 onnx>=1.7.0
-# Align with upstream PyTorch requirements
-protobuf~=5.29.5
```

Comment:
Are we sure that this is necessary? I'm reluctant to remove since I don't know why this is here.